atomic

```diff
@@ -411,6 +415,7 @@ def create_parser(atomic):
     scanp.add_argument("--scan_type", default=None, help=_("define the intended scanner"))
     scanp.add_argument("--list", action='store_true', default=False, help=_("List available scanners"))
     scanp.add_argument("--verbose", action='store_true', default=False, help=_("Show more output from scanning container"))
+    scanp.add_argument("--default_scanner", action='store', default=default_scanner, help=argparse.SUPPRESS)
     scan_group.add_argument("--rootfs", nargs='?', default=[], action='append', help=_("Rootfs path to scan"))
```

Comment:
Further explanation ...  

1.  As of today, if there is no default scanner declared, any subcommand of atomic including scan will fail with an error message pointing that out.  Obviously that can be fixed.
2. With the plan to have the scanner images themselves deliver the configuration file to /etc/atomic.d/ through the the install command, there will be no automatic editing of /etc/atomic.conf to set a default scanner.
3. Therefore, my feeling was that if there was only *one* scanner installed, it could become the "default" even though it was not declared so in /etc/atomic.conf.
4. The argparse option for default_scanner is hidden and never exposed to the users.  It is populated with a default scanner name based on the following:
  * if a default scanner is declared in /etc/atomic.conf, it is the default  
  * if only one scanner is installed, it becomes the default.
  * if more than one scanner is installed but not default defined in /etc/atomic.conf, it errors out with a meaningful message to set the default.